src/main/resources/static/css/output.css

```diff
@@ -510,6 +510,10 @@ Ensure the default browser behavior of the `hidden` attribute.
   visibility: hidden;
 }
 
+.static{
```

Comment:
Probably need to regenerate as I don't think this is used anywhere

Reply:
regenerated & also tried manually deleting, but tailwind seems persistent on having this in the output file...

Reply:
🤷‍♂️